tools/distpartitioning/utils.py

```diff
@@ -67,6 +67,28 @@ def read_json(json_file):
 
     return val
 
+def get_etype_featnames(etype_name, schema_map):
+    """
+    Retrieves edge feature names for a given edge_type
```

Comment:
Let's keep the consistency, we usually don;t change line here.

"""Retrieves edge feature names for a given edge_type.

Reply:
Done.